test/plugins/templates.test.js

```diff
@@ -153,7 +153,7 @@ describe('template plugin test', () => {
         beforeEach(() => {
             options = {
                 method: 'GET',
-                url: '/templates/screwdriver%2Fbuild/1.7.3'
```

Comment:
If a user wants to input their template name `chef/publish` in the url, do they need to use `%2F`?